Tools/build/check_warnings.py

```diff
@@ -11,13 +11,15 @@
 from pathlib import Path
 from typing import NamedTuple
 
+
 class FileWarnings(NamedTuple):
     name: str
     count: int
 
 
 def extract_warnings_from_compiler_output_clang(
     compiler_output: str,
+    path_prefix: str | None = None,
```

Comment:
How it's used now `path_prefix` can't be `None`, maybe should be `""` as a default?

Reply:
Switched to default to `""` which is better I agree